java/test/src/main/java/io/ray/test/PlacementGroupTest.java

```diff
@@ -224,4 +228,21 @@ public void testCompatibleForPreviousApi() {
     Assert.assertNotNull(resPlacementGroup);
     Assert.assertEquals(resPlacementGroup.getBundles().size(), 1);
   }
+
+  @Test(groups = {"cluster"})
+  public void testPlacementGroupForNormalTask() {
+    PlacementGroup placementGroup = PlacementGroupTestUtils.createSimpleGroup();
+    Assert.assertTrue(placementGroup.wait(60));
+
+    Assert.assertEquals(
+        Ray.task(Counter::ping)
+            .setPlacementGroup(placementGroup, 0)
+            .setResource("CPU", 1.0)
+            .remote()
+            .get(),
+        "pong");
+
+    // Make sure it will not affect the previous normal task.
+    Assert.assertEquals(Ray.task(Counter::ping).remote().get(), "pong");
```

Comment:
How do you know that the two normal tasks are scheduled on different nodes?

Reply:
hm... actually, I think there is no need to verify the correctness of the placement logic since it has been verified in the python API  `test_placement_group.py`. I think the idea from @jovany-wang  is better, we can submit a task that required a non-exist placement group resource and make sure its scheduling will fail (or timeout), what do you think?